eventfiles/templates/file/event_files.html

```diff
@@ -0,0 +1,58 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <title>Event files</title>
+        {% load static %}
```

Comment:
can you use the base HTML template.

In your case, you need to remove the HTML, head, and body tags because they are already inside the base HTML template.

An example can be found here:
https://github.com/redhat-beyond/lets-meet/blob/04f6afac8f75c3ca2d24a701b9f2aee9feed885b/users/templates/update_profile/update_form.html

Reply:
Done